lib/universal/git-hooks.ts

```diff
@@ -0,0 +1,108 @@
+
+import { parse } from "https://deno.land/std/flags/mod.ts";
+import { Command } from "https://deno.land/x/cliffy/command/mod.ts";
```

Comment:
Be sure to turn on Deno linting in VS Code, it will remind you that this is not a proper way to reference imports. You should use pinned versions.